pass-journal-loader-nih/src/main/java/org/dataconservancy/pass/loader/journal/nih/LoaderEngine.java

```diff
@@ -119,39 +136,63 @@ private void load(Journal j, boolean doUpdates) {
             } catch (final Exception e) {
                 LOG.warn("Could not load journal " + j.getName(), e);
             }
-        } else if (doUpdates &&
-                ((found.getPmcParticipation() != j.getPmcParticipation()) || !found.getIssns().containsAll(j
-                        .getIssns()))) {
+        } else if (doUpdates) {
 
-            // IF PMC participation or ISSNs has changed, update the journal in the repository.
             try {
+                boolean update = false;
                 final Journal toUpdate = client.readResource(found.getId(), Journal.class);
-                toUpdate.setIssns(j.getIssns());
 
-                if (j instanceof PMCSource) {
+                if (toUpdate.getPmcParticipation() != j.getPmcParticipation()) {
```

Comment:
So on this line here, can a medline record (which has no PMC data) clobber the value of any pre-existing journal record's PMC participation?  The point of PMCSource was a marker to distinguish data that contains relevant PMC participation data vs data that doesn't.

Reply:
well, since only pmc sourced data is allowed to update, a medline entry can't. but, a new complication is going to have us allow medline data to perform updates as well - we will need to change semantics so that hasPmcParticimation in the call to load() from main is no longer a proxy for doUpdates, and instead really means that the source has PMC participation info. if we change the update logic to reflect this new reality, then we might be able to get rid of the marker class, since the boolean passed in as hasPmcParticipation and interpreted as doUpdates can now be interpreted as hasPmcParticipation - this will be true for journals read in from PMC, and fals for Medline

Reply:
i will work on the new wrinkle, and apply to the PR

Reply:
I see.  Yeah, if we can get rid of that class that easily, then I'd say go for it